src/main/java/hudson/plugins/git/GitStatus.java

```diff
@@ -313,6 +325,54 @@ public void writeBody(PrintWriter w) {
         }
 
         /**
+         * Check if there is a ParameterValue with the same name
+         *
+         */
+        private boolean parameterValueExist (List<ParameterValue> parameterValues, String name) {
+            for (ParameterValue parameterValue : parameterValues) {
+                if (parameterValue.getName().equals(name)) {
+                    return true;
+                }
+            }
+            return false;
+        }
+
+        /**
+         * Get the default parameters values from a job
+         *
+         */
+        private List<ParameterValue> getDefaultParametersValues(Job job) {
+            ArrayList<ParameterValue> defValues;
```

Comment:
You have an implicit assumption by the caller of this method that it is returned a mutable list, so change the return type to `ArrayList`